libgdiplus fails to build from source with GCC 15 on amd64

The package libgdiplus with version 6.1-r3 fails to build from source with GCC 15 on amd64 with the following error:

```2025/05/31 00:13:34 INFO -- Detecting C compile features - done
2025/05/31 00:13:34 INFO -- Detecting CXX compiler ABI info
2025/05/31 00:13:34 INFO /bin/sh ../libtool  --tag=CC   --mode=link x86_64-pc-linux-gnu-gcc  -g -O2 -pthread   -o testlineargradientbrush testlineargradientbrush.o ../src/libgdiplus.la -lm -lglib-2.0 -lcairo -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif
2025/05/31 00:13:34 INFO libtool: link: x86_64-pc-linux-gnu-gcc -g -O2 -pthread -o .libs/testmatrix testmatrix.o  ../src/.libs/libgdiplus.so -lm -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif -pthread
2025/05/31 00:13:34 INFO libtool: link: x86_64-pc-linux-gnu-gcc -g -O2 -pthread -o .libs/testlineargradientbrush testlineargradientbrush.o  ../src/.libs/libgdiplus.so -lm -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif -pthread
2025/05/31 00:13:34 INFO -- Detecting CXX compiler ABI info - done
2025/05/31 00:13:34 INFO -- Check for working CXX compiler: /usr/local/bin/c++ - skipped
2025/05/31 00:13:34 INFO -- Detecting CXX compile features
2025/05/31 00:13:34 INFO -- Detecting CXX compile features - done
2025/05/31 00:13:34 WARN CMake Deprecation Warning at googlemock/CMakeLists.txt:45 (cmake_minimum_required):
2025/05/31 00:13:34 WARN   Compatibility with CMake < 3.10 will be removed from a future version of
2025/05/31 00:13:34 WARN   CMake.
2025/05/31 00:13:34 WARN
2025/05/31 00:13:34 WARN   Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
2025/05/31 00:13:34 WARN   to tell CMake that the project requires at least <min> but has been updated
2025/05/31 00:13:34 WARN   to work with policies introduced by <max> or earlier.
2025/05/31 00:13:34 WARN
2025/05/31 00:13:34 WARN
2025/05/31 00:13:34 WARN CMake Deprecation Warning at googletest/CMakeLists.txt:56 (cmake_minimum_required):
2025/05/31 00:13:34 WARN   Compatibility with CMake < 3.10 will be removed from a future version of
2025/05/31 00:13:34 WARN   CMake.
2025/05/31 00:13:34 WARN
2025/05/31 00:13:34 WARN   Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
2025/05/31 00:13:34 WARN   to tell CMake that the project requires at least <min> but has been updated
2025/05/31 00:13:34 WARN   to work with policies introduced by <max> or earlier.
2025/05/31 00:13:34 WARN
2025/05/31 00:13:34 WARN
2025/05/31 00:13:34 WARN CMake Warning (dev) at googletest/cmake/internal_utils.cmake:243 (find_package):
2025/05/31 00:13:34 WARN   Policy CMP0148 is not set: The FindPythonInterp and FindPythonLibs modules
2025/05/31 00:13:34 WARN   are removed.  Run "cmake --help-policy CMP0148" for policy details.  Use
2025/05/31 00:13:34 WARN   the cmake_policy command to set the policy and suppress this warning.
2025/05/31 00:13:34 WARN
2025/05/31 00:13:34 WARN Call Stack (most recent call first):
2025/05/31 00:13:34 WARN   googletest/CMakeLists.txt:91 (include)
2025/05/31 00:13:34 WARN This warning is for project developers.  Use -Wno-dev to suppress it.
2025/05/31 00:13:34 WARN
2025/05/31 00:13:34 INFO /bin/sh ../libtool  --tag=CC   --mode=link x86_64-pc-linux-gnu-gcc  -g -O2 -pthread   -o testjpegcodec testjpegcodec.o ../src/libgdiplus.la -lm -lglib-2.0 -lcairo -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif
2025/05/31 00:13:34 INFO -- Found PythonInterp: /usr/bin/python3.13 (found version "3.13.3")
2025/05/31 00:13:34 INFO -- Performing Test CMAKE_HAVE_LIBC_PTHREAD
2025/05/31 00:13:34 INFO /bin/sh ../libtool  --tag=CC   --mode=link x86_64-pc-linux-gnu-gcc  -g -O2 -pthread   -o testpen testpen.o ../src/libgdiplus.la -lm -lglib-2.0 -lcairo -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif
2025/05/31 00:13:34 INFO libtool: link: x86_64-pc-linux-gnu-gcc -g -O2 -pthread -o .libs/testjpegcodec testjpegcodec.o  ../src/.libs/libgdiplus.so -lm -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif -pthread
2025/05/31 00:13:34 INFO -- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
2025/05/31 00:13:34 INFO -- Found Threads: TRUE
2025/05/31 00:13:34 INFO -- Configuring done (0.9s)
2025/05/31 00:13:34 INFO libtool: link: x86_64-pc-linux-gnu-gcc -g -O2 -pthread -o .libs/testpen testpen.o  ../src/.libs/libgdiplus.so -lm -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif -pthread
2025/05/31 00:13:34 INFO -- Generating done (0.0s)
2025/05/31 00:13:34 INFO -- Build files have been written to: /home/build/gtest_build
2025/05/31 00:13:34 INFO make[3]: Entering directory '/home/build/gtest_build'
2025/05/31 00:13:34 INFO [ 12%] Building CXX object googletest/CMakeFiles/gtest.dir/src/gtest-all.cc.o
2025/05/31 00:13:34 INFO /bin/sh ../libtool  --tag=CC   --mode=link x86_64-pc-linux-gnu-gcc  -g -O2 -pthread   -o testpathgradientbrush testpathgradientbrush.o ../src/libgdiplus.la -lm -lglib-2.0 -lcairo -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif
2025/05/31 00:13:34 INFO /bin/sh ../libtool  --tag=CC   --mode=link x86_64-pc-linux-gnu-gcc  -g -O2 -pthread   -o testtiffcodec testtiffcodec.o ../src/libgdiplus.la -lm -lglib-2.0 -lcairo -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif
2025/05/31 00:13:34 INFO libtool: link: x86_64-pc-linux-gnu-gcc -g -O2 -pthread -o .libs/testpathgradientbrush testpathgradientbrush.o  ../src/.libs/libgdiplus.so -lm -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif -pthread
2025/05/31 00:13:34 INFO libtool: link: x86_64-pc-linux-gnu-gcc -g -O2 -pthread -o .libs/testtiffcodec testtiffcodec.o  ../src/.libs/libgdiplus.so -lm -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif -pthread
2025/05/31 00:13:34 INFO /bin/sh ../libtool  --tag=CC   --mode=link x86_64-pc-linux-gnu-gcc  -g -O2 -pthread   -o testgifcodec testgifcodec.o ../src/libgdiplus.la -lm -lglib-2.0 -lcairo -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif
2025/05/31 00:13:34 INFO libtool: link: x86_64-pc-linux-gnu-gcc -g -O2 -pthread -o .libs/testgifcodec testgifcodec.o  ../src/.libs/libgdiplus.so -lm -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif -pthread
2025/05/31 00:13:34 INFO /bin/sh ../libtool  --tag=CC   --mode=link x86_64-pc-linux-gnu-gcc  -g -O2 -pthread   -o testwmfcodec testwmfcodec.o ../src/libgdiplus.la -lm -lglib-2.0 -lcairo -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif
2025/05/31 00:13:35 INFO /bin/sh ../libtool  --tag=CC   --mode=link x86_64-pc-linux-gnu-gcc  -g -O2 -pthread   -o testtexturebrush testtexturebrush.o ../src/libgdiplus.la -lm -lglib-2.0 -lcairo -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif
2025/05/31 00:13:35 INFO libtool: link: x86_64-pc-linux-gnu-gcc -g -O2 -pthread -o .libs/testwmfcodec testwmfcodec.o  ../src/.libs/libgdiplus.so -lm -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif -pthread
2025/05/31 00:13:35 INFO libtool: link: x86_64-pc-linux-gnu-gcc -g -O2 -pthread -o .libs/testtexturebrush testtexturebrush.o  ../src/.libs/libgdiplus.so -lm -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif -pthread
2025/05/31 00:13:35 INFO /bin/sh ../libtool  --tag=CC   --mode=link x86_64-pc-linux-gnu-gcc  -g -O2 -pthread   -o testgraphicspath testgraphicspath.o ../src/libgdiplus.la -lm -lglib-2.0 -lcairo -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif
2025/05/31 00:13:35 INFO libtool: link: x86_64-pc-linux-gnu-gcc -g -O2 -pthread -o .libs/testgraphicspath testgraphicspath.o  ../src/.libs/libgdiplus.so -lm -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif -pthread
2025/05/31 00:13:35 INFO /bin/sh ../libtool  --tag=CC   --mode=link x86_64-pc-linux-gnu-gcc  -g -O2 -pthread   -o testgraphicsfill testgraphicsfill.o ../src/libgdiplus.la -lm -lglib-2.0 -lcairo -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif
2025/05/31 00:13:35 INFO /bin/sh ../libtool  --tag=CC   --mode=link x86_64-pc-linux-gnu-gcc  -g -O2 -pthread   -o testgpimage testgpimage.o ../src/libgdiplus.la -lm -lglib-2.0 -lcairo -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif
2025/05/31 00:13:35 INFO /bin/sh ../libtool  --tag=CC   --mode=link x86_64-pc-linux-gnu-gcc  -g -O2 -pthread   -o testpngcodec testpngcodec.o ../src/libgdiplus.la -lm -lglib-2.0 -lcairo -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif
2025/05/31 00:13:35 INFO libtool: link: x86_64-pc-linux-gnu-gcc -g -O2 -pthread -o .libs/testgraphicsfill testgraphicsfill.o  ../src/.libs/libgdiplus.so -lm -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif -pthread
2025/05/31 00:13:35 INFO libtool: link: x86_64-pc-linux-gnu-gcc -g -O2 -pthread -o .libs/testgpimage testgpimage.o  ../src/.libs/libgdiplus.so -lm -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif -pthread
2025/05/31 00:13:35 INFO libtool: link: x86_64-pc-linux-gnu-gcc -g -O2 -pthread -o .libs/testpngcodec testpngcodec.o  ../src/.libs/libgdiplus.so -lm -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif -pthread
2025/05/31 00:13:35 INFO /bin/sh ../libtool  --tag=CC   --mode=link x86_64-pc-linux-gnu-gcc  -g -O2 -pthread   -o testgraphics testgraphics.o ../src/libgdiplus.la -lm -lglib-2.0 -lcairo -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif
2025/05/31 00:13:35 INFO libtool: link: x86_64-pc-linux-gnu-gcc -g -O2 -pthread -o .libs/testgraphics testgraphics.o  ../src/.libs/libgdiplus.so -lm -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif -pthread
2025/05/31 00:13:35 INFO /bin/sh ../libtool  --tag=CC   --mode=link x86_64-pc-linux-gnu-gcc  -g -O2 -pthread   -o testemfcodec testemfcodec.o ../src/libgdiplus.la -lm -lglib-2.0 -lcairo -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif
2025/05/31 00:13:35 INFO /bin/sh ../libtool  --tag=CC   --mode=link x86_64-pc-linux-gnu-gcc  -g -O2 -pthread   -o testbitmap testbitmap.o ../src/libgdiplus.la -lm -lglib-2.0 -lcairo -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif
2025/05/31 00:13:35 INFO libtool: link: x86_64-pc-linux-gnu-gcc -g -O2 -pthread -o .libs/testemfcodec testemfcodec.o  ../src/.libs/libgdiplus.so -lm -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif -pthread
2025/05/31 00:13:35 INFO libtool: link: x86_64-pc-linux-gnu-gcc -g -O2 -pthread -o .libs/testbitmap testbitmap.o  ../src/.libs/libgdiplus.so -lm -lpangocairo-1.0 -lpango-1.0 -lcairo -lharfbuzz -lgobject-2.0 -lglib-2.0 -lfontconfig -lfreetype -ljpeg -L/usr -ltiff -lgif -lpng16 -lX11 -lexif -pthread
2025/05/31 00:13:35 WARN In file included from /home/build/external/googletest/googletest/src/gtest-all.cc:42:
2025/05/31 00:13:35 WARN /home/build/external/googletest/googletest/src/gtest-death-test.cc: In function 'pid_t testing::internal::ExecDeathTestSpawnChild(char* const*, int)':
2025/05/31 00:13:35 WARN /home/build/external/googletest/googletest/src/gtest-death-test.cc:1385:26: error: 'uintptr_t' does not name a type
2025/05/31 00:13:35 WARN  1385 |         reinterpret_cast<uintptr_t>(stack_top) % kMaxStackAlignment == 0);
2025/05/31 00:13:35 WARN       |                          ^~~~~~~~~
2025/05/31 00:13:35 WARN /home/build/external/googletest/googletest/src/gtest-death-test.cc:308:38: note: in definition of macro 'GTEST_DEATH_TEST_CHECK_'
2025/05/31 00:13:35 WARN   308 |     if (!::testing::internal::IsTrue(expression)) { \
2025/05/31 00:13:35 WARN       |                                      ^~~~~~~~~~
2025/05/31 00:13:35 WARN /home/build/external/googletest/googletest/src/gtest-death-test.cc:49:1: note: 'uintptr_t' is defined in header '<cstdint>'; this is probably fixable by adding '#include <cstdint>'
2025/05/31 00:13:35 WARN    48 | # include <limits.h>
2025/05/31 00:13:35 WARN   +++ |+#include <cstdint>
2025/05/31 00:13:35 WARN    49 | 
2025/05/31 00:13:36 WARN make[5]: *** [googletest/CMakeFiles/gtest.dir/build.make:79: googletest/CMakeFiles/gtest.dir/src/gtest-all.cc.o] Error 1
2025/05/31 00:13:36 WARN make[4]: *** [CMakeFiles/Makefile2:199: googletest/CMakeFiles/gtest.dir/all] Error 2
2025/05/31 00:13:36 INFO make[3]: Leaving directory '/home/build/gtest_build'
2025/05/31 00:13:36 WARN make[3]: *** [Makefile:146: all] Error 2
2025/05/31 00:13:36 WARN make[2]: *** [Makefile:1984: ../gtest_build/.stamp-gtest] Error 2
2025/05/31 00:13:36 WARN make[2]: *** Waiting for unfinished jobs....
2025/05/31 00:13:44 INFO make[2]: Leaving directory '/home/build/tests'
2025/05/31 00:13:44 INFO make[1]: Leaving directory '/home/build'
2025/05/31 00:13:44 WARN make[1]: *** [Makefile:490: all-recursive] Error 1
2025/05/31 00:13:44 WARN make: *** [Makefile:397: all] Error 2
2025/05/31 00:13:44 INFO make: Leaving directory '/home/build'
2025/05/31 00:13:45 ERRO failed to build package: unable to run package libgdiplus pipeline: unable to run pipeline: unable to run pipeline: exit status 2
make[1]: *** [Makefile:157: packages/x86_64/libgdiplus-6.1-r3.apk] Error 1
make[1]: Leaving directory '/home/sergio_durigan_chainguard_dev/os'
make: *** [Makefile:151: package/libgdiplus] Error 2
```

--- a/libgdiplus.yaml
+++ b/libgdiplus.yaml
@@ -1,7 +1,7 @@
 package:
   name: libgdiplus
   version: 6.2
-  epoch: 1
+  epoch: 2
   description: "Open Source Implementation of the GDI+ API"
   copyright:
     - license: MIT
@@ -18,7 +18,6 @@ environment:
       - cmake
       - expat-dev
       - fribidi-dev
-      - gcc-14-default
       - giflib-dev
       - glib-dev
       - glib-static

--- a/libical.yaml
+++ b/libical.yaml
@@ -2,7 +2,7 @@
 package:
   name: libical
   version: 3.0.20
-  epoch: 2
+  epoch: 3
   description: Reference implementation of the iCalendar format
   copyright:
     - license: LGPL-2.1-only OR MPL-2.0
@@ -16,7 +16,6 @@ environment:
       - busybox
       - ca-certificates-bundle
       - cmake
-      - gcc-14-default
       - glib-dev
       - glib-gir
       - gobject-introspection-dev

--- a/libtirpc.yaml
+++ b/libtirpc.yaml
@@ -1,7 +1,7 @@
 package:
   name: libtirpc
   version: "1.3.7"
-  epoch: 0
+  epoch: 1
   description: Transport Independent RPC library (SunRPC replacement)
   copyright:
     - license: BSD-3-Clause
@@ -14,7 +14,6 @@ environment:
       - build-base
       - busybox
       - ca-certificates-bundle
-      - gcc-14-default
       - krb5-dev
       - libtool
 

--- a/malcontent.yaml
+++ b/malcontent.yaml
@@ -1,7 +1,7 @@
 package:
   name: malcontent
   version: "1.15.1"
-  epoch: 0 # CVE-2025-47907
+  epoch: 1
   description: enumerate file capabilities, including malicious behaviors
   copyright:
     - license: Apache-2.0
@@ -13,7 +13,6 @@ package:
 environment:
   contents:
     packages:
-      - gcc-14-default
       - openssl-dev
       - yara-x
 

--- a/mariadb-12.0.yaml
+++ b/mariadb-12.0.yaml
@@ -1,7 +1,7 @@
 package:
   name: mariadb-12.0
   version: "12.0.2"
-  epoch: 0
+  epoch: 1
   description: "The MariaDB open source relational database"
   copyright:
     - license: GPL-3.0-or-later
@@ -22,7 +22,6 @@ environment:
       - busybox
       - ca-certificates-bundle
       - cmake
-      - gcc-14-default
       - libaio-dev
       - libevent-dev
       - linux-pam-dev

--- a/mariadb-connector-c.yaml
+++ b/mariadb-connector-c.yaml
@@ -1,7 +1,7 @@
 package:
   name: mariadb-connector-c
   version: "3.4.7"
-  epoch: 0
+  epoch: 1
   description: The MariaDB Native Client library (C driver)
   copyright:
     - license: LGPL-2.1-or-later
@@ -15,7 +15,6 @@ environment:
       - busybox
       - ca-certificates-bundle
       - cmake
-      - gcc-14-default
       - openssl-dev
       - samurai
       - zlib-dev

--- a/mysql-connector-cpp.yaml
+++ b/mysql-connector-cpp.yaml
@@ -1,7 +1,7 @@
 package:
   name: mysql-connector-cpp
   version: "9.4.0"
-  epoch: 0
+  epoch: 1
   description: MySQL Connector/C++ is a MySQL database connector for C++
   copyright:
     - license: GPL-3.0-or-later
@@ -17,7 +17,6 @@ environment:
       - busybox
       - ca-certificates-bundle
       - cmake-3
-      - gcc-14-default
       - lz4-dev
       - openssl-dev
       - protobuf-dev

--- a/nginx-mainline.yaml
+++ b/nginx-mainline.yaml
@@ -3,7 +3,7 @@ package:
   # MAINLINE VERSIONS MUST USE ODD MINOR VERSIONS (e.g., 1.27.x, 1.29.x)
   # Must also bump njs at the same time
   version: "1.29.1"
-  epoch: 1
+  epoch: 2
   description: HTTP and reverse proxy server (mainline version)
   copyright:
     - license: BSD-2-Clause
@@ -26,7 +26,6 @@ environment:
       - build-base
       - busybox
       - ca-certificates-bundle
-      - gcc-14-default
       - gd-dev
       - geoip-dev
       - libmaxminddb-dev

--- a/ninja-build.yaml
+++ b/ninja-build.yaml
@@ -9,7 +9,7 @@
 package:
   name: ninja-build
   version: "1.13.1"
-  epoch: 1
+  epoch: 2
   description: "Ninja is a small build system with a focus on speed."
   copyright:
     - license: Apache-2.0
@@ -20,7 +20,6 @@ environment:
       - build-base
       - busybox
       - ca-certificates-bundle
-      - gcc-14-default
       - wolfi-base
 
 pipeline:

--- a/ollama.yaml
+++ b/ollama.yaml
@@ -1,7 +1,7 @@
 package:
   name: ollama
   version: "0.11.7"
-  epoch: 0
+  epoch: 1
   description: Get up and running with Llama 2 and other large language models locally
   copyright:
     - license: MIT
@@ -9,7 +9,6 @@ package:
 environment:
   contents:
     packages:
-      - gcc-14-default
 
 pipeline:
   - uses: git-checkout

--- a/pstack.yaml
+++ b/pstack.yaml
@@ -1,7 +1,7 @@
 package:
   name: pstack
   version: "2.14"
-  epoch: 0
+  epoch: 1
   description: "Print stack traces from running processes, or core files."
   copyright:
     - license: BSD-2-Clause
@@ -11,7 +11,6 @@ environment:
     packages:
       - build-base
       - cmake
-      - gcc-14-default
       - python-3.12
       - wolfi-base
       - xz-dev

--- a/py3-sentencepiece.yaml
+++ b/py3-sentencepiece.yaml
@@ -1,7 +1,7 @@
 package:
   name: py3-sentencepiece
   version: "0.2.1"
-  epoch: 0
+  epoch: 1
   description: SentencePiece is an unsupervised text tokenizer and detokenizer
   copyright:
     - license: Apache-2.0
@@ -23,7 +23,6 @@ environment:
       - build-base
       - busybox
       - cmake
-      - gcc-14-default
       - py3-supported-pip
       - py3-supported-python-dev
 

--- a/py3-tensorflow-data-validation.yaml
+++ b/py3-tensorflow-data-validation.yaml
@@ -1,7 +1,7 @@
 package:
   name: py3-tensorflow-data-validation
   version: "1.17.0"
-  epoch: 1
+  epoch: 2
   description: Library for exploring and validating machine learning data
   copyright:
     - license: Apache-2.0
@@ -30,7 +30,6 @@ environment:
       - build-base
       - busybox
       - ca-certificates-bundle
-      - gcc-14-default
       - numpy
       - openjdk-11
       - patch

--- a/python-3.10.yaml
+++ b/python-3.10.yaml
@@ -1,7 +1,7 @@
 package:
   name: python-3.10
   version: "3.10.18"
-  epoch: 4
+  epoch: 5
   description: "the Python programming language"
   copyright:
     - license: PSF-2.0
@@ -20,7 +20,6 @@ environment:
       - bzip2-dev
       - ca-certificates-bundle
       - expat-dev
-      - gcc-14-default
       - gdbm-dev
       - libffi-dev
       - linux-headers

